packages/components/src/components/combobox/combobox.styles.ts

```diff
@@ -0,0 +1,344 @@
+    padding-block: var(--syn-spacing-x-small);
+    padding-inline: 0;
+    overflow: auto;
+    overscroll-behavior: none;
+
+    /* Make sure it adheres to the popup's auto size */
+    max-width: var(--auto-size-available-width);
+    max-height: var(--auto-size-available-height);
+  }
+
+  .combobox__listbox ::slotted(syn-divider) {
+    --spacing: var(--syn-spacing-x-small);
+  }
+
+  .combobox__listbox ::slotted(small) {
+    display: block;
+    font-size: var(--syn-font-size-small);
+    font-weight: var(--syn-font-weight-semibold);
+    color: var(--syn-color-neutral-500);
+    padding-block: var(--syn-spacing-2x-small);
+    padding-inline: var(--syn-spacing-x-large);
+  }
+
+
+  /**
+   * Invalid user data
+   */
+  :host([data-user-invalid]) .combobox__inputs {
+    border-color: var(--syn-input-border-color-focus-error);
+  }
+
+  :host([data-user-invalid]) .combobox--standard:not(.combobox--disabled).combobox--open .combobox__inputs,
+  :host([data-user-invalid]) .combobox--standard:not(.combobox--disabled).combobox--focused .combobox__inputs {
+    border-color: var(--syn-input-border-color-focus-error);
+    box-shadow: 0 0 0 var(--syn-focus-ring-width) var(--syn-input-focus-ring-error);
+  }
+
+  /* Clear button */
+  .combobox__clear {
+    color: var(--syn-input-icon-icon-clearable-color);
+    font-size: var(--syn-spacing-large);
+  }
+
+  .combobox--small .combobox__clear {
+    font-size: var(--syn-spacing-medium);
+  }
+
+  .combobox--large .combobox__clear {
+    font-size: var(--syn-spacing-x-large);
+  }
+
+  .combobox--medium .combobox__clear {
+    margin-inline-start: var(--syn-spacing-small);
+  }
+
+  /* Expand icon */
+  .combobox__expand-icon {
+    color: var(--syn-color-neutral-950);
+    font-size: var(--syn-spacing-large);
+    margin-inline-start: var(--syn-spacing-small);
+  }
+
+  /* Change combobox border on hover */
+  .combobox:not(.combobox--disabled):hover .combobox__inputs {
+    border-color: var(--syn-input-color-hover);
+  }
+
+  /* Prefix and Suffix */
+
+  /* Small */
+  .combobox--small .combobox__prefix::slotted(*) {
+    margin-inline-end: var(--syn-spacing-x-small);
+  }
+
+  .combobox--small .combobox__suffix::slotted(*) {
+    margin-inline-start: var(--syn-spacing-x-small);
+  }
+
+  .combobox--medium .combobox__prefix::slotted(*) {
+    margin-inline-end: var(--syn-input-spacing-small);
+  }
+
+  .combobox--medium .combobox__suffix::slotted(*) {
+    margin-inline-start: var(--syn-input-spacing-small);
+  }
+
+  .combobox--large .combobox__prefix::slotted(*) {
+    margin-inline-end: var(--syn-input-spacing-medium);
+  }
+
+  .combobox--large .combobox__suffix::slotted(*) {
+    margin-inline-start: var(--syn-input-spacing-medium);
+  }
+
+  .combobox__prefix,
+  .combobox__suffix {
+    color: var(--syn-input-help-text-color);
+  }
+
+  /* Listbox */
+  .combobox__listbox {
+    /* @todo: Should be --syn-border-radius-medium, which should be set to 0 */
```

Comment:
Just had a look, it seems it is set to 0 (`--syn-input-border-radius-medium: var(--syn-border-radius-none);`). Maybe you could use the real token here now?

Reply:
Not the `--syn-input-border-radius-medium` token but `--syn-border-radius-medium` is used here (it is not the input element itself, but the open list underneath the input).
And this token is still not set to 0: 
![Bildschirmfoto 2024-09-09 um 13 39 10](https://github.com/user-attachments/assets/fd4b3a74-8207-41d1-b51f-9d4e236410c8)

Reply:
Ah, got it. Why don´t we use the input-border-radius here? Or just use `syn-border-radius-none` directly?

Reply:
good question. I can change it to use the syn-border-radius-none instead for the select, so we get it also in the combobox.
But I would let the comment stay there, because if at some point we use the correct token, this can be used from others to better adapt the theme (if they want to use rounded border)